db/migrate/20200319134722_create_articles.rb

```diff
@@ -0,0 +1,11 @@
+class CreateArticles < ActiveRecord::Migration[6.0]
+  def change
+    create_table :articles do |t|
+      t.string :title
+      t.text :teaser
+      t.text :content
+
```

Comment:
do we get id added to db column once we use serializer?

Reply:
Rails gives auto Id to the items check the response file we sent to you earlier while we had the meeting: (tagged you)